lesson_1_8.py

```diff
@@ -0,0 +1,2 @@
+a = int(input("Введите количество рядов: "))
+for i in range(1, a + 1): print(" " * (a - i), "*" * (i + i - 1))
```

Comment:
Такая запись хоть и разрешена, но не принята
```suggestion
for i in range(1, a + 1):
    print(" " * (a - i), "*" * (i + i - 1))
```

Reply:
У вас лишний пробел. Как думаете, почему?
![image](https://github.com/Leodgar2010/python_improved_homework/assets/49741194/2c1f9779-0359-474f-afc5-745e357f38af)